wallaby.js

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+module.exports = wallaby => {
```

Comment:
I'm on the fence about whether we should have configuration for a paid product in the top level of the library, what if instead you create a gist of this configuration, and we add a testing section to the README.md that could link to it?

Reply:
Done. I have removed the file from this PR and created https://gist.github.com/safarmer/7d112dd6e22bcf96286623b06e7191c3